modules/gdnative/include/gdnative/math_defs.h

```diff
@@ -36,10 +36,11 @@ extern "C" {
 #endif
 
 #include <stdint.h>
-
+#include <stdboo.h>
+	
 ////// bool
 
-typedef bool godot_bool;
+typedef _Bool godot_bool;
```

Comment:
As far as I understand `stdbool.h` defines `bool`, so we should be able to keep using `bool` as is.

Reply:
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/stdbool.h.html

Reply:
It looks like this in GCC:
```
#ifndef _STDBOOL_H
#define _STDBOOL_H

#ifndef __cplusplus

#define bool    _Bool
#define true    1
#define false   0
...
```

BTW @vnen we could maybe drop `GODOT_TRUE` and `GODOT_FALSE` defines if we include `stdbool.h` which defines `true` and `false`. Though that's material for another PR, and should likely not be cherry-picked to `3.2` so we don't break compat where not needed.

Reply:
Sorry about the typo, I edited this on the web :-)

Replaced as suggested with the original `bool`